snippets/inkscape-1.0.1/textext-1.1.0/inkscape/06/config.json

```diff
@@ -0,0 +1,24 @@
+{
+	"description": "transformations in Inkscape are preserved across editing",
+	"original" : {
+		"text" : "$$\\lim_{x \\to 0} \\frac{\\sin(x)}{x} = 1$$",
+		"scale-factor" : 1.0,
+		"preamble-file" : null,
+		"pdf_to_png" : "convert"
+	},
+	"modified" : {
+		"text" : "$$\\lim_{y \\to 0} \\frac{\\sin(y)}{y} = 1$$",
+		"scale-factor" : 2.715,
+		"preamble-file" : null
+	},
+	"check" : {
+		"render" : {
+			"scale-is-preserved": true,
+			"render-area": "document",
+			"dpi": 300
+		},
+		"compare" : {
+			"fuzz" : "100%"
```

Comment:
Why fuzz is 100%?

Reply:
This is `nonsense="100%"` of course. I fixed it.